integration-tests/api/helpers/draft-order-seeder.js

```diff
@@ -23,13 +24,22 @@ module.exports = async (connection, data = {}) => {
     type: "default",
   })
 
+  const salesChannel = await simpleSalesChannelFactory(connection, {
+    id: "sales-channel",
+    is_default: true,
+  })
+
   await manager.insert(Product, {
     id: "test-product",
     title: "test product",
     profile_id: defaultProfile.id,
     options: [{ id: "test-option", title: "Size" }],
   })
 
+  await manager.query(
```

Comment:
suggestion: should it be part of the product factory? we pass a sales channel and it creates the pivot along it if it is not managed by the ORM